### DISCUSSION TITLE
1.2.10

### DISCUSSION THREAD

**Discussion by trueromanus** [anilibria/anilibria-winmaclinux#149] [Анонсы]
- Исправлена ошибка приводящая к тому что при пропуске опенинга позиция возвращалась обратно
- Исправлена ошибка при переходе на страницу через меню с открытой карточкой карточка остается открытой
- Настройки соединения с удаленным плеером перенесены в общие настройки
- Страница TorrentStream теперь выгружаемая
- Добавлена ссылка на инструкцию по TorrentStream для случая когда не удалось подключиться
- Добавлен автоподключение TorrentStream для случая когда он запускается вне приложения
- Добавлена поддержка внешнего плеера и первая интеграция rutube 
- Добавлена поддержка настройки страниц левого верхнего тулбара

<hr /><em>This discussion was created from the release <a href='https://github.com/anilibria/anilibria-winmaclinux/releases/tag/1.2.10'>1.2.10</a>.</em>

**Comment by X1mper** [comment]
Собрал обновление. Завтра будет доступно в ALT Linux Sisyphus (Нестабильном репозитории):
https://packages.altlinux.org/ru/tasks/330832/

**Reply by trueromanus** [reply]
Спасибо за Ваш вклад!

**Comment by mr9733n** [comment]
anilibria_windows_1.2.10.zip
https://www.virustotal.com/gui/file/89ec80bbef8112c909b91c4d3c9855a0cd84487b02e5ce60582a777312c2f7d5/detection

**Reply by trueromanus** [reply]
Отлично теперь Вы знаете какой антивирус точно не стоит ставить и тем более покупать :) А если серьезно то доверять (т.е. ставить) или нет решать лично Вам. Я никакие вирусы в свои программы не добавляю но доверять мне на слово Вам конечно же не стоит.
